diff --git a/testsuite-10.1.0/src/test_lib.C b/testsuite-10.1.0/src/test_lib.C
index 3808c79..73d5dff 100644
--- a/testsuite-10.1.0/src/test_lib.C
+++ b/testsuite-10.1.0/src/test_lib.C
@@ -110,7 +110,7 @@ LocErr::LocErr(const char *__file__, const int __line__, const std::string msg)
 	line__(__line__)
 {}
 
-LocErr::~LocErr() THROW
+LocErr::~LocErr() throw()
 {}
 
 std::string LocErr::file() const
diff --git a/testsuite-10.1.0/src/test_lib.h b/testsuite-10.1.0/src/test_lib.h
index d2a1940..b37676d 100644
--- a/testsuite-10.1.0/src/test_lib.h
+++ b/testsuite-10.1.0/src/test_lib.h
@@ -156,7 +156,7 @@ class LocErr
 			const int __line__,
 			const std::string msg); 
 
-	TESTLIB_DLL_EXPORT virtual ~LocErr() THROW; 
+	TESTLIB_DLL_EXPORT virtual ~LocErr() throw();
 
 	TESTLIB_DLL_EXPORT std::string file() const;
 
-- 
2.23.0

